doc/examples/reconst_qtiplus.py

```diff
@@ -0,0 +1,263 @@
+Instructions on how to install cvxpy
+can be found at https://www.cvxpy.org/install/. A free SDP solver
+called 'SCS' is installed with cvxpy, and can be readily used for
+performing the fit. However, it is recommended to install an
+alternative solver, MOSEK, for improved speed and performance.
+MOSEK requires a licence which is free for academic use.
+Instructions on how to install Mosek and setting up a licence can be found
+at https://docs.mosek.com/latest/install/installation.html
+
+Usage example
+==============
+
+Here we show how metrics derived from the
+QTI model parameters compares when the fit is performed with and without
+applying the positivity constraints.
+
+In DIPY, the constrained estimation routine is avaiable as part of the
+`dipy.reconst.qti` module.
+First we import all the necessary modules to perform the QTI fit:
+"""
+
+from dipy.data import read_DiB_217_lte_pte_ste, read_DiB_70_lte_pte_ste
+import dipy.reconst.qti as qti
+
+"""
+To showcase why enforcing positivity constraints in QTI is relevant, we use
+a human brain dataset comprising 70 volumes acquired with tensor-encoding.
+This dataset was obtained by subsampling a richer dataset containing 217
+diffusion measurements, which we will use as ground truth when comparing
+the parameters estimation with and without applied constraints. This emulates
+performing short data acquisition which can correspond to scanning patients
+in clinical settings.
+
+The full dataset used in this tutorial was originally published at
+https://github.com/filip-szczepankiewicz/Szczepankiewicz_DIB_2019,
+and is described in [3]_.
+
+"""
+
+"""
+First, let's load the complete dataset and create the gradient table.
+We mark these two with the '_217' suffix.
+
+"""
+
+data_img, mask_img, gtab_217 = read_DiB_217_lte_pte_ste()
+data_217 = data_img.get_fdata()
+mask = mask_img.get_fdata()
+
+"""
+Second, let's load the downsampled dataset and create the gradient table.
+We mark these two with the '_70' suffix.
+
+"""
+
+data_img, _, gtab_70 = read_DiB_70_lte_pte_ste()
+data_70 = data_img.get_fdata()
+
+"""
+Now we can fit the QTI model to the datasets containing 217 measurements, and
+use it as reference to compare the constrained and unconstrained fit on the
+smaller dataset. For time considerations, we restrict the fit to a
+single slice.
+"""
+
+mask[:, :, :13] = 0
+mask[:, :, 14:] = 0
+
+qtimodel_217 = qti.QtiModel(gtab_217)
+qtifit_217 = qtimodel_217.fit(data_217, mask)
+
+
+"""
+Now we can fit the QTI model using the default unconstrained fitting method
+to the subsampled dataset:
+"""
+
+qtimodel_unconstrained = qti.QtiModel(gtab_70)
+qtifit_unconstrained = qtimodel_unconstrained.fit(data_70, mask)
+
+"""
+Now we repeat the fit but with the constraints applied.
+To perform the constrained fit, we select the 'SDPdc' fit method when creating
+the QtiModel object.
+Note: this fit method is slower compared to the defaults unconstrained.
```

Comment:
Instead of the classic way, it would be good to do the foolowing:

```python
.. note::
    this fit method is slower compared to the defaults unconstrained.
```